src/Core/Application/Service/TaskExecutor.php

```diff
@@ -20,14 +20,14 @@
 class TaskExecutor
 {
     /**
-     * @var \SprykerSdk\Sdk\Core\Application\Service\PlaceholderResolver
+     * @var string
      */
-    protected PlaceholderResolver $placeholderResolver;
+    public const DYNAMIC_TASK = 'dynamic';
```

Comment:
1. For constants in Spryker Yoda naming type is preferred. So it should be `TASK_DYNAMIC`
2. Is it used somewhere?

Reply:
removed